src/App.tsx

```diff
@@ -1,85 +1,135 @@
 import React from 'react';
 import 'bulma/css/bulma.css';
 import './App.scss';
+import classNames from 'classnames';
 
 export const goods = [
-  'Dumplings',
-  'Carrot',
-  'Eggs',
-  'Ice cream',
-  'Apple',
-  'Bread',
-  'Fish',
-  'Honey',
-  'Jam',
-  'Garlic',
+  {
+    name: 'Dumplings',
+    id: 1,
```

Comment:
No need to create array of objects here, especially manually
Your products are already unique, and if you want to be sure that nothing duplicated will be added in future, you can set to key attribute string that consist from product name + it's index